context_adapter.go

```diff
@@ -14,12 +14,7 @@
 
 package gormadapter
```

Comment:
Update the README: https://github.com/casbin/gorm-adapter?tab=readme-ov-file#context-adapter